src/app/ranking/page.tsx

```diff
@@ -0,0 +1,79 @@
+import styles from "./ranking.module.css";
+import Image from "next/image";
+import data from "../../data/currentRanking.json";
+import { Fragment } from "react";
+
```

Comment:
You can add the font here since it's only use on this page at the moment : 
```suggestion
import {VT323} from "next/font/google";

const font = VT323({
    subsets: ['latin'],
    weight: "400"
})
```
ℹ️   You'll just have to add `next/fonts` to the package.json 👌 
Looks like next handle fonts smoothly : [doc](https://beta.nextjs.org/docs/optimizing/fonts#google-fonts)